api/turing/config/testdata/config-2.yaml

```diff
@@ -27,6 +27,3 @@ Experiment:
         quxkey2-3: quxval2-3-add
     quux: 
       quuxkey1: quuxval1
-OpenapiConfig:
-  SwaggerUIConfig:
-    ServingPath: /swagger-ui
```

Comment:
Why remove this?

Reply:
Leaving this thread as the central discussion point: https://github.com/gojek/turing/pull/140#discussion_r760959454